src/plugins/dashboard/public/dashboard_container/embeddable/api/overlays/save_modal.tsx

```diff
@@ -114,8 +114,10 @@ export class DashboardSaveModal extends React.Component<Props, State> {
               defaultMessage="Description"
             />
           }
+          fullWidth
```

Comment:
This PR looks really cool!

Can you explain these Dashboard save modal changes? I can't see any noticeable difference in the actual Dashboard.

Reply:
Ahh, you beat me to the review :)

I noticed that it only seems to make a difference in mobile view, but it honestly looks a bit weird either way because the tags dropdown isn't full width:

- **Before**

    ![image](https://github.com/elastic/kibana/assets/8698078/a181bd8b-8b4e-4237-a099-d0bc8efaedb9)

- **After**

    ![image](https://github.com/elastic/kibana/assets/8698078/41a7e24a-d1e8-4ebf-9b9e-2b5853db54d2)


The save modal we are using is deprecated anyway and will have to be replaced at some point on the dashboard side, so I'm honestly fine either way. But was also curious why it was added

Reply:
Hey! I think y'all were so fast on the draw that you caught me in the middle of the design changes. Could you check again? As far as I can see, things are resolved...
<img width="819" alt="Screenshot 2023-05-26 at 4 40 53 PM" src="https://github.com/elastic/kibana/assets/315764/3bedab5b-7c7d-4da8-833f-89bb77df5481">